misc/toml-parser/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -0,0 +1,228 @@
+            "name": "KeyValue",
+            "base": "ModuleMemberDeclarationNode",
+            "kind": "KEY_VALUE",
+            "attributes": [
+                {
+                    "name": "identifier",
+                    "type": "KeyNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                },
+                {
+                    "name": "assign",
+                    "type": "Token",
+                    "isOptional": false
+                },
+                {
+                    "name": "value",
+                    "type": "ValueNode",
+                    "isOptional": false
+                }
+            ]
+        },
+        {
+            "name": "Array",
+            "base": "ValueNode",
+            "kind": "ARRAY",
+            "attributes": [
+                {
+                    "name": "openBracket",
+                    "type": "Token",
+                    "isOptional": false
+                },
+                {
+                    "name": "values",
+                    "type": "ValueNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                },
+                {
+                    "name": "closeBracket",
+                    "type": "Token",
+                    "isOptional": false
+                }
+            ]
+        },
+        {
+            "name": "StringLiteralNode",
+            "base": "ValueNode",
+            "kind": "STRING_LITERAL",
+            "attributes": [
+                {
+                    "name": "startDoubleQuote",
+                    "type": "Token"
+                },
+                {
+                    "name": "content",
+                    "type": "Token"
+                },
+                {
+                    "name": "endDoubleQuote",
+                    "type": "Token"
+                }
+            ]
+        },
+        {
+            "name": "IntLiteralNode",
+            "base": "ValueNode",
+            "kind": "DEC_INT",
+            "attributes": [
+                {
+                    "name": "value",
+                    "type": "Token",
+                    "isOptional": false
+                }
+            ]
+        },
+        {
+            "name": "FloatLiteralNode",
+            "base": "ValueNode",
+            "kind": "FLOAT",
+            "attributes": [
+                {
+                    "name": "value",
+                    "type": "Token",
+                    "isOptional": false
+                }
+            ]
+        },
+        {
+            "name": "BoolLiteralNode",
+            "base": "ValueNode",
+            "kind": "BOOLEAN",
+            "attributes": [
+                {
+                    "name": "value",
+                    "type": "Token",
+                    "isOptional": false
+                }
+            ]
+        },
+        {
+            "name": "UnquotedLiteralNode",
+            "base": "ValueNode",
+            "kind": "UNQUOTED_KEY_TOKEN",
```

Comment:
Shall we call this an `IdentifierLiteralNode`?

Also, the kind should reflect the node type. Having "token" suffix to a non-terminal node is misleading.